### PR TITLE
Feature/add params for frame

### LINKED ISSUE #326: Add parameters to configure area_description and start_of_service frames id

### PR DESCRIPTION
**Pull request by PerrineAguiar**
Fixes #326.

### CODE REVIEW

File: tango_ros_common/tango_ros_native/include/tango_ros_native/tango_3d_reconstruction_helper.h [hunk 1]
```diff
@@ -54,11 +54,13 @@ void UpdateMesh(const Tango3DR_ReconstructionContext& t3dr_context,
 void ExtractMeshAndConvertToMarkerArray(
     const Tango3DR_ReconstructionContext& t3dr_context,
     const Tango3DR_GridIndexArray& t3dr_updated_indices,
-    double time_offset, visualization_msgs::MarkerArray* mesh_marker_array);
+    double time_offset, const std::string& frame_id,
```
Comment (smits) [inline tango_ros_common/tango_ros_native/include/tango_ros_native/tango_3d_reconstruction_helper.h]: I would rename this to base_frame_id, for consistency

File: tango_ros_common/tango_ros_native/include/tango_ros_native/tango_3d_reconstruction_helper.h [hunk 2]
```diff
@@ -54,11 +54,13 @@ void UpdateMesh(const Tango3DR_ReconstructionContext& t3dr_context,
...
+    visualization_msgs::MarkerArray* mesh_marker_array);
 
 bool ExtractFloorPlanImageAndConvertToOccupancyGrid(
     const Tango3DR_ReconstructionContext& t3dr_context,
-    double time_offset, double t3dr_resolution, uint8_t threshold,
+    double time_offset, const std::string& frame_id,
```
Comment (smits) [inline tango_ros_common/tango_ros_native/include/tango_ros_native/tango_3d_reconstruction_helper.h]: Here as well.

File: tango_ros_common/tango_ros_native/include/tango_ros_native/tango_ros_conversions_helper.h [hunk 3]
```diff
@@ -117,7 +119,7 @@ void toTango3DR_Pose(const TangoPoseData& tango_pose_data, Tango3DR_Pose* t3dr_p
 // @param mesh_marker, the output visualization_msgs::Marker.
 void toMeshMarker(const Tango3DR_GridIndex& grid_index,
                   const Tango3DR_Mesh& tango_mesh,
-                  double time_offset,
+                  double time_offset, const std::string& frame_id,
```
Comment (smits) [inline tango_ros_common/tango_ros_native/include/tango_ros_native/tango_ros_conversions_helper.h]: Here as well

File: tango_ros_common/tango_ros_native/include/tango_ros_native/tango_ros_conversions_helper.h [hunk 4]
```diff
@@ -133,9 +135,9 @@ void toMeshMarker(const Tango3DR_GridIndex& grid_index,
 // Pixel value > threshold --> cell is occupied.
 // @param occupancy_grid, the output nav_msgs::OccupancyGrid.
 void toOccupancyGrid(const Tango3DR_ImageBuffer& image_grid,
-                     const Tango3DR_Vector2& origin,
-                     double time_offset, double resolution, uint8_t threshold,
-                     nav_msgs::OccupancyGrid* occupancy_grid);
+                     const Tango3DR_Vector2& origin, double time_offset,
+                     const std::string& frame_id, double resolution,
```
Comment (smits) [inline tango_ros_common/tango_ros_native/include/tango_ros_native/tango_ros_conversions_helper.h]: and here

File: tango_ros_common/tango_ros_native/src/tango_3d_reconstruction_helper.cpp [hunk 5]
```diff
@@ -200,7 +200,8 @@ void ExtractMeshAndConvertToMarkerArray(
 
 bool ExtractFloorPlanImageAndConvertToOccupancyGrid(
     const Tango3DR_ReconstructionContext& t3dr_context,
-    double time_offset, double t3dr_resolution, uint8_t threshold,
+    double time_offset, const std::string& frame_id,
```
Comment (smits) [inline tango_ros_common/tango_ros_native/src/tango_3d_reconstruction_helper.cpp]: here as well

File: tango_ros_common/tango_ros_native/src/tango_3d_reconstruction_helper.cpp [hunk 6]
```diff
@@ -165,7 +165,8 @@ void UpdateMesh(const Tango3DR_ReconstructionContext& t3dr_context,
 void ExtractMeshAndConvertToMarkerArray(
     const Tango3DR_ReconstructionContext& t3dr_context,
     const Tango3DR_GridIndexArray& t3dr_updated_indices,
-    double time_offset, visualization_msgs::MarkerArray* mesh_marker_array) {
+    double time_offset, const std::string& frame_id,
```
Comment (smits) [inline tango_ros_common/tango_ros_native/src/tango_3d_reconstruction_helper.cpp]: and here

File: tango_ros_common/tango_ros_native/src/tango_ros_conversions_helper.cpp [hunk 7]
```diff
@@ -212,9 +214,9 @@ void toTango3DR_Pose(const TangoPoseData& tango_pose_data, Tango3DR_Pose* t3dr_p
 
 void toMeshMarker(const Tango3DR_GridIndex& grid_index,
                   const Tango3DR_Mesh& tango_mesh,
-                  double time_offset,
+                  double time_offset, const std::string& frame_id,
```
Comment (smits) [inline tango_ros_common/tango_ros_native/src/tango_ros_conversions_helper.cpp]: and here

**Comment by PerrineAguiar** [comment]
@smits Thanks for your review.

**Pull request merged**